spec/support/page_helpers.rb

```diff
@@ -1,49 +1,17 @@
 # frozen_string_literal: true
 
 module PageHelpers
-  # rubocop:disable Metrics/CyclomaticComplexity
-  # rubocop:disable Metrics/MethodLength
-  # rubocop:disable Metrics/PerceivedComplexity
-  # rubocop:disable Metrics/ParameterLists
-  def reload_page_until_timeout!(text:, as_link: false, table: nil, with_reindex: false, with_events_expanded: false,
-                                 selector: nil)
+  def reload_page_until_timeout!(text: '')
```

Comment:
The way this method now works: either give me text, and I'll check the page for it, or give me a block and I'll yield to the consumer to do what it needs on each reload.